packages/cfd/src/Components/Routes/binary-routes.jsx

```diff
@@ -19,8 +19,8 @@ const BinaryRoutes = props => {
             }}
         >
             <Switch>
-                {getRoutesConfig({ is_dashboard }).map(route => (
-                    <RouteWithSubRoutes key={route.path} {...route} {...props} />
+                {getRoutesConfig({ is_dashboard }).map((route, idx) => (
+                    <RouteWithSubRoutes key={idx} {...route} {...props} />
```

Comment:
I have used route.path as a key since it is the only unique thing in the getRoutesConfig but we still get the warning that key needs to be unique. I need to use index as the key or use uuid to create unique keys.

Reply:
Is there any other value to use beside `idx`? 
Ref: https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
The only usable thing is path. What about adding a hard-coded `id` to the getRoutesConfig of cfd package (id: 1 for first route and id: 2 for second route) ?

Reply:
can you check where there are duplicate paths? It shouldn't be the case

Reply:
Yes better check for duplicate paths.